apps/socketoptions.hpp

```diff
@@ -247,15 +247,18 @@ const SocketOption srt_options [] {
     { "ipv6only", 0, SRTO_IPV6ONLY, SocketOption::PRE, SocketOption::INT, nullptr },
     { "peeridletimeo", 0, SRTO_PEERIDLETIMEO, SocketOption::PRE, SocketOption::INT, nullptr },
     { "packetfilter", 0, SRTO_PACKETFILTER, SocketOption::PRE, SocketOption::STRING, nullptr },
+    { "sender", 0, SRTO_SENDER, SocketOption::PRE, SocketOption::INT, nullptr },
```

Comment:
Why is it necessary for AEAD? Does AEAD also work in bidirectional mode?

Reply:
Was added by mistake.